BSP/Inc/Interrupt_Priorities.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2018-2022 UT Longhorn Racing Solar */
+/* Interrupt_Priorities.h
+ * File to define Preemptive and Sub Interrupt Priorities.
```

Comment:
This needs to be more descriptive. Imagine the questions you had when you started this project. What layer is this used for? When should this be updated? Any considerations to keep in mind when changing values.

Reply:
I'm not sure what to do about "when should this be updated", but I added some more that explained what each one does